src/SDKs/ServiceBus/Management.ServiceBus/Microsoft.Azure.Management.ServiceBus.csproj

```diff
@@ -7,18 +7,27 @@
 		<PackageId>Microsoft.Azure.Management.ServiceBus</PackageId>
 		<Description>Provides developers with libraries to create and manage Namespaces and manage Authorization Rules. Note: This client library is for ServiceBus under Azure Resource Manager.</Description>
 		<AssemblyName>Microsoft.Azure.Management.ServiceBus</AssemblyName>
-		<Version>1.0.2</Version>
+		<Version>1.0.3</Version>
 		<PackageTags>Microsoft Azure ServiceBus Management;ServiceBus;ServiceBus management;</PackageTags>
 		<PackageReleaseNotes>
 			<![CDATA[
-		This change adds metadata Geo-Disaster recovery support to the Service Bus .NET management client.
-		1) Added APIs for creating and managing Disaster recovery configurations which allows users to pair two Service Bus namespaces in a primary-secondary configuration and creates an alias to represent the DR pair.
-		2) Added APIs to trigger a Geo-DR failover which re-points the alias from the primary to the secondary namespace and APIs to break DR pairing which disables Geo-DR
+		      Servicebus: Geo DR Update (Alias) - 
+          1) Authorization rule related new API for Geo DR (Alias) are added 
+          2) AlternateName property is added to ArmDisasterRecovery
+
+          Servicebus: AutoForward for Queue and Subscription - 
+          1) added 2 properties added to Queue and subscription
+              i) ForwardTo
+             ii) ForwardDeadLetteredMessagesTo
 	  ]]>
 		</PackageReleaseNotes>
 		<TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
 	</PropertyGroup>
 
+  <PropertyGroup>
```

Comment:
@v-Ajnava you will not need this if you do the following:
1 Pull latest from upstream
2 execute msbuild build.proj from your local repo root
3 msbuild build.proj /t:Build /p:Scope=SDKs\ServiceBus
this should take care of that error

Reply:
Resolved